src/PostgREST/Config.hs

```diff
@@ -395,6 +394,13 @@ parser optPath env dbSettings roleSettings roleIsolationLvl =
         Nothing   -> pure Nothing
         Just orig -> pure $ Just (T.strip <$> T.splitOn "," orig)
 
+    parseDbSchemas :: C.Key -> C.Key -> C.Parser C.Config (NonEmpty Text)
+    parseDbSchemas k al =
+      optWithAlias (optString k) (optString al) >>= \case
+        Nothing      -> pure $ fromList ["public"]
+        Just ""      -> fail "Invalid db-schemas. Check your configuration."
+        Just schemas -> pure (fromList $ T.strip <$> T.splitOn "," schemas)
```

Comment:
@wolfgangwalther Can you help double check if this is the correct behavior? :pray: